lib/build/source-inclusion.js

```diff
@@ -27,8 +28,16 @@ exports.SourceInclusion = class {
     this.items.push(item);
   }
 
+  isExcluded(item){
+	  let found = this.excludes.findIndex(x => {			
+			let exclusion = new this.bundle.Minimatch(x);
```

Comment:
Sounds good.







> On Sep 2, 2016, at 3:21 PM, Andrew Stoker <notifications@github.com> wrote:
> 
> 
> In lib/build/source-inclusion.js
> :
> 
>> @@ -27,8 +28,16 @@ exports.SourceInclusion = class {
>>      this.items.push(item);
>>    }
>>  
>> +  isExcluded(item){
>> +	  let found = this.excludes.findIndex(x => {			
>> +			let exclusion = new this.bundle.Minimatch(x);
> I was thinking about that, but wasn't set on exactly where to place it. I gotta drive home from work early to avoid the flash floods here, so I'll find a more efficient place for it and have it in in an hour or so.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> , or mute the thread
> .<embedded image>
> 
>